docker-compose.yml

```diff
@@ -203,7 +203,7 @@ services:
     volumes:
       - ./app/editor/src:/usr/app/src
       - ./app/editor/public:/usr/app/public
-      - tno-editor-node-cache:/usr/app/node_modules
+      # - tno-editor-node-cache:/usr/app/node_modules
```

Comment:
As far as I can tell yarn v2 doesn't need this.  It throws errors if I leave it here so this has to be removed.